src/Aquifer.Public.API/Endpoints/Bibles/Texts/Get/Response.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Aquifer.Public.API.Endpoints.Bibles.Texts.Get;
+
+public class Response
+{
+    public required int BibleId { get; set; }
+    public required string BibleName { get; set; }
+    public required string BibleAbbreviation { get; set; }
+    public required string BookName { get; set; }
+    public required string BookCode { get; set; }
+    public required IReadOnlyList<ResponseChapter> Chapters { get; set; }
```

Comment:
I'm a big fan of `IReadOnlyList<T>` over `ICollection<T>`.  It's a lightweight wrapper that communicates to other devs that the list should not be mutated elsewhere and you can arrays and `List<T>`s to it.

`ReadOnlySet<T>` is coming soon in .net 9.

Reply:
Yeah, this sounds good to me.